src/learning-catalogue/model/course.ts

```diff
@@ -43,7 +44,7 @@ export class Course {
 		groups: ['all', 'status'],
 		message: 'course.validation.status.invalid',
 	})
-	status: Status
+	status: Status = Status.DRAFT
```

Comment:
this was making me a bit uncomfortable before - the fact that any `Course` object constructed will have `DRAFT` status - that means if we load an object and status for some reason doesn't come through it will be set to `DRAFT`. Just 2p.